core/metacling/src/TCling.cxx

```diff
@@ -1258,9 +1290,37 @@ TCling::TCling(const char *name, const char *title)
       // Load libc and stl first.
       LoadModules({"libc", "stl"}, *fInterpreter);
 
-      LoadModules({"ROOT_Foundation_C", "ROOT_Config", "ROOT_Foundation_Stage1_NoRTTI", "Core", "RIO"}, *fInterpreter);
-      if (!fromRootCling)
-         LoadModules({"TreePlayer", "ROOTVecOps"}, *fInterpreter);
+      // Load core modules
+      std::vector<std::string> CoreModules = {"ROOT_Foundation_C","ROOT_Config","ROOT_Foundation_Stage1_NoRTTI", "Core", "RIO"};
+      // Exclude buggy modules
```

Comment:
What is buggy about these modules? What are they excluded from? Why do we build them if we cannot use them? Could you be more specific?